ansible/playbooks/backup-volumes.yml

```
@@ -0,0 +1,110 @@
+---
+- name: Docker Volume Backup
+  hosts: "{{ target_hosts | default('enabled_hosts') }}"
+  gather_facts: yes
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)